### PR TITLE
Adds the "Try Portus" and "Feedback, comments, contributions?" sections.

### PR DESCRIPTION
**Pull request by eotchi**
Adding two additional sections 
* Try Portus
* Feedback, comments, contributions?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -98,6 +98,23 @@
         changes.
       </p>
 
+      <h2>
+        <a id="audit" class="anchor" href="#audit" aria-hidden="true"><span class="octicon octicon-link"></span></a>
+        Try Portus
+      </h2>
+
+      <p>
+        Portus is available on <a href="https://github.com/SUSE/Portus"> github</a>, but also as a ready to go <a href="https://github.com/SUSE/Portus/wiki/Installing-Portus"> appliance</a> based on openSUSE. <br>
```
Comment (mssola) [inline index.html]: - github -> Github
- The appliance link should be https://github.com/SUSE/Portus/wiki/Installing-Portus#the-appliance

File: index.html [hunk 2]
```diff
@@ -98,6 +98,23 @@
...
+        All Portus documentation, including the installation guide is available in our <a href="https://github.com/SUSE/Portus/wiki"> wiki</a>
```
Comment (mssola) [inline index.html]: You forgot to add a "." after "wiki" ;)

File: index.html [hunk 3]
```diff
@@ -98,6 +98,23 @@
...
+        Portus is available on <a href="https://github.com/SUSE/Portus"> github</a>, but also as a ready to go <a href="https://github.com/SUSE/Portus/wiki/Installing-Portus"> appliance</a> based on openSUSE. <br>
+        All Portus documentation, including the installation guide is available in our <a href="https://github.com/SUSE/Portus/wiki"> wiki</a> 
+      </p>
+
+      <h2>
+        <a id="audit" class="anchor" href="#audit" aria-hidden="true"><span class="octicon octicon-link"></span></a>
+        Feedback, comments, contributions? 
+      </h2>
+      <p>
+        Open an <a href="https://github.com/SUSE/Portus/issues"> issue</a> or subscribe to the <a href="http://lists.suse.com/mailman/listinfo/containers">containers@lists.suse.com</a> mailing list
```
Comment (mssola) [inline index.html]: You forgot a "." at the end.

**Comment by mssola** [comment]
Nitpicking, sorry :) Besides that, it looks good to me

File: index.html [hunk 4]
```diff
@@ -98,6 +98,23 @@
         changes.
       </p>
 
+      <h2>
+        <a id="audit" class="anchor" href="#audit" aria-hidden="true"><span class="octicon octicon-link"></span></a>
+        Try Portus
+      </h2>
+
+      <p>
+        Portus is available on <a href="https://github.com/SUSE/Portus"> github</a>, but also as a ready to go <a href="https://github.com/SUSE/Portus/wiki/Installing-Portus#the-appliance"> appliance</a> based on openSUSE. <br>
```
Comment (mssola) [inline index.html]: Nitpicking, sorry, but please: github -> GitHub (it's how their [official name](https://github.com/about) is written after all)

**Comment by flavio** [comment]
LGTM

**Comment by mssola** [comment]
LGTM

**Pull request merged**